appeals/api/src/server/endpoints/appeals/appeals.formatter.js

```diff
@@ -277,4 +288,28 @@ export const mapAppealToDueDate = (appeal, appellantCaseStatus, appellantCaseDue
 	}
 };
 
-export { formatAppeal, formatAppeals, formatMyAppeals };
+/**
+ * @param {AppealRelationship[]} linkedAppeals
+ * @param {string} currentAppealRef
+ * @returns {number[]}
+ */
+const getRelevantLinkedAppealIds = (linkedAppeals, currentAppealRef) => {
+	const isParentAppeal = linkedAppeals.some((link) => link.parentRef === currentAppealRef);
+
+	console.log('getRelevantLinkedAppealIds', JSON.stringify(linkedAppeals));
```

Comment:
we should get rid of console.logs - if we need to persist logs use the logger (pino)